### PR TITLE
data/systemd: tweak comment

### PR DESCRIPTION
**Pull request by zyga**
Signed-off-by: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>

### CODE REVIEW

File: data/systemd/Makefile [hunk 1]
```diff
@@ -53,8 +53,8 @@ clean:
 		sed s:@SNAP_MOUNT_DIR@:$(SNAP_MOUNT_DIR):g  | \
 		cat > $@
 
-# If SNAP_MOUNT_DIR uses non-default location then just copy the file.
-# This lets us reuse the sed rule above and get the right dependencies.
+# If SNAP_MOUNT_DIR uses non-default location then rename snap.mount
+# so that the directory name is encoded in the file name.
```
Comment (bboozzoo) [inline data/systemd/Makefile]: The comment is a bit too easy to miss. Can't we always generate snap.mount and then copy it under the proper name in the `install` target?

Comment (zyga) [reply data/systemd/Makefile]: That would be a bit more verbose (filter out and separate install) but I think so, sure.

Comment (bboozzoo) [reply data/systemd/Makefile]: Sorry, I completely misread the code.

Comment (zyga) [reply data/systemd/Makefile]: No worries, if you think that it should be that way I can tweak it further.

**Pull request merged**